client/app/bundles/Restaurants/constants/restaurantsConstants.jsx

```diff
@@ -1,3 +1,6 @@
 /* eslint-disable import/prefer-default-export */
 
 export const HELLO_WORLD_NAME_UPDATE = 'HELLO_WORLD_NAME_UPDATE';
```

Comment:
I like the constants file.
But I think the colors can go to the CSS later., no?